decidim-participatory_processes/app/views/layouts/decidim/_process_header_steps.html.erb

```diff
@@ -23,7 +23,7 @@
       </div>
       <% cta_text = translated_attribute(participatory_process.active_step.cta_text) %>
       <% if participatory_process.active_step.cta_path.present? && cta_text.present? %>
-        <%= link_to t(".view_steps"), decidim_participatory_processes.participatory_process_participatory_process_steps_path(current_participatory_space) %>
+        <%= link_to t(".view_steps"), decidim_participatory_processes.participatory_process_participatory_process_steps_path(current_participatory_space), :class => "clear button secondary tiny show-for-medium" %>
```

Comment:
💬 Sorry for being so nitpicky, but can you change this using the new syntax please (`class: "clear ..."`). Thanks!